templates/info.html

```diff
@@ -0,0 +1,432 @@
+<div class="px-1 py-1 pb-md-4 mx-auto">
```

Comment:
I notice that you have created a `div` for every idea. In such cases, we will have to update both the YAML and the HTML file whenever ideas need to be changed. It would be better rather to have a for loop running over the parsed YAML object and generate these div tags for every idea in the YAML file. 

Something like:
```
{% for idea in ideas %}
<div class="idea">
....    
</div>
{% endfor %}
```
You may need to change the format in which you pass the information you send to the renderer. The idea is to have a single page where you just need to update the YAML file and the rendering for the updated ideas happens on it's own. If we have to update the HTML everytime, there is no use of having Jinja2 based rendering.